src/main/java/Group15/View/MuscleSelectionView.java

```diff
@@ -48,7 +48,7 @@ public static Scene createMuscleSelectorScene() {
 
         Button backButton = new Button("Back");
         backButton.setPrefSize(200, 50);
-        backButton.setOnAction(_ -> ViewController.goBack());
+        backButton.setOnAction(_ -> ViewController.setScene(HomeScreenView.createScene()));
```

Comment:
You dont have to use the setscene here. The only way to get into MuscleSelection is from the homescreen. GoBack method is sufficient